test/dlc_tests/container_tests/bin/testBackendWithKeras

```diff
@@ -54,4 +53,4 @@ fi
 set +e
 rm -rf ${HOME_DIR}/.keras/datasets
 set -e
-exit 0
```

Comment:
nit: It should be better to keep a newline at the end of a file

Reply:
added empty line